git-extra/git-process.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Check if git is running
+# -x flag only match processes whose name (or command line if -f is specified) exactly match the pattern. 
+if pgrep -x "git-bash.exe" > /dev/null
```

Comment:
This would catch _all_ `git-bash.exe` processes, even the ones from portable Gits that are not at all affected by the installer.